beams/app/model/fit.py

```diff
@@ -779,19 +849,28 @@ def lambdify(expression: str, variables=None):
     var_names = [INDEPENDENT_VARIABLE] if INDEPENDENT_VARIABLE in expression_string else []
     var_names.extend([_replace_unsupported_unicode_characters(var) for var in variables])
 
-    lambda_expression = sp.lambdify(var_names, sp.sympify(expression_string), ["numpy", "scipy"])
+    if expression_string.split('(')[0] in external_function_dict:
+        return external_function_dict[expression_string.split('(')[0]]
+
+    # TODO: Ask Alec if this is okay
+    if ALPHA in var_names:
+        var_names.remove(ALPHA)
```

Comment:
![suspicious-eyes](https://user-images.githubusercontent.com/42580833/222477475-162bb6bd-1e9d-4487-b2be-72082f11d95c.gif)

Reply:
Does it work? lol